### DISCUSSION TITLE
建议插件cluster-key-rate-limit 可以增加,不限时的次数限流规则

### DISCUSSION THREAD

**Discussion by MrOldPeng** [alibaba/higress#1276] [Ideas]
| query_per_second | int    | 否，`query_per_second`,`query_per_minute`,`query_per_hour`,`query_per_day` 中选填一项 | -      | 允许每秒请求次数                                             |
| query_per_minute | int    | 否，`query_per_second`,`query_per_minute`,`query_per_hour`,`query_per_day` 中选填一项 | -      | 允许每分钟请求次数                                           |
| query_per_hour   | int    | 否，`query_per_second`,`query_per_minute`,`query_per_hour`,`query_per_day` 中选填一项 | -      | 允许每小时请求次数                                           |
| query_per_day    | int    | 否，`query_per_second`,`query_per_minute`,`query_per_hour`,`query_per_day` 中选填一项 | -      | 允许每天请求次数                                             |

现在最多只允许设置每天的请求次数,其实可以增加个,不限时,但是限制次数的规则

**Comment by johnlanni** [comment]
是什么样的场景有这样的需求呢？
